Makefile

```diff
@@ -1,11 +1,16 @@
 CC=clang
 CXX=clang++
-LLVM_CONFIG=llvm-config
+LLVM_CONFIG=llvm-config-11
```

Comment:
Any reason for the `-11` suffix? FWIW, I only have `llvm-config` on my machine!

Reply:
I moved the code to bencher14 and Debian supports multiple versions of llvm so they add that suffix, but I thought I force pushed to remove that before opening the pr.

Reply:
OK, I think let's add a commit to just go back to `llvm-config` then?

Reply:
fixed